src/pages/ProfileSettings.vue

```diff
@@ -77,4 +83,14 @@ q-page.fit.row.wrap.justify-center
   width: 100%
   max-width: 30rem
   height: 100%
+
+.edit-btn
+  left: 210px
+
+.btn-width
+  width: 350px
+  justify-center: center
+  font-size: 20px
+  background-color: orange
+  border-radius: 5px
 </style>
```

Comment:
Please move these changes to a separate branch

Reply:
Noted.